os/drivers/audio/alc5658.c

```diff
@@ -154,12 +154,18 @@ struct sample_rate_entry_s {
 };
 
 static const struct sample_rate_entry_s g_sample_entry[] = {
-	{AUDIO_SAMP_RATE_8K, codec_init_pll_8K, sizeof(codec_init_pll_8K)},
-	{AUDIO_SAMP_RATE_11K, codec_init_pll_11K, sizeof(codec_init_pll_11K)},
-	{AUDIO_SAMP_RATE_16K, codec_init_pll_16K, sizeof(codec_init_pll_16K)},
-	{AUDIO_SAMP_RATE_22K, codec_init_pll_22K, sizeof(codec_init_pll_22K)},
-	{AUDIO_SAMP_RATE_32K, codec_init_pll_32K, sizeof(codec_init_pll_32K)},
-	{AUDIO_SAMP_RATE_44K, codec_init_pll_44K, sizeof(codec_init_pll_44K)},
+	{AUDIO_SAMP_RATE_8K, codec_init_pll_8K, sizeof(codec_init_pll_8K)}
```

Comment:
Coding style here changed many times. like a toggling.
What is correct??

Reply:
I reverted this change cuz the previous seems better.